README.md

```diff
@@ -192,6 +192,35 @@ A test case, to test this plugin looks like this:
   function named `plugin_name`. As such, this line tests whether `host` is returned when the plugin function
   `hostname` is called with the parameter `fqdn`.
 
+## References
+
+To use pytest-inmanta to test code using References, nothing special is required when using the `deploy_resource_*` endpoints
+
+However, when inspecting individual resources, some care is required. 
+1. After the `project.compile` call, all attributes containing reference will be `null`
+2. To get the correct value, use `project.resolve_references(resource)` this will update the resource in-place
```

Comment:
```suggestion
1. After the `project.compile` call, all attributes containing reference will be `null`.
2. To get the correct value, use `project.resolve_references(resource)`. This will update the resource in-place.
```